source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/with-data.tsx

```diff
@@ -2,11 +2,14 @@ import * as React from 'react';
 import { compose } from 'recompose';
 import { withData as withOrbit, WithDataProps } from 'react-orbitjs';
 
-import { query, defaultSourceOptions } from '@data';
+import { query, defaultSourceOptions, defaultOptions } from '@data';
```

Comment:
defaultSourceOptions is no longer require

Reply:
both are used